tests/e2e/specs/Pools.spec.js

```diff
@@ -228,4 +230,29 @@ describe('Pools', () => {
 
     getElementAtPosition(startEventPosition, nodeTypes.startEvent).should('exist');
   });
+
+  it('should not cover child elements with lane', function() {
+    const poolPosition = { x: 300, y: 300 };
+    dragFromSourceToDest(nodeTypes.pool, poolPosition);
+
+    getElementAtPosition(poolPosition, nodeTypes.pool)
+      .click()
+      .then($pool => {
+        getCrownButtonForElement($pool, 'lane-above-button').click({ force: true });
+      });
+
+    const startEventPosition = { x: 150, y: 150 };
+    const topLanePosition = { x: startEventPosition.x, y: startEventPosition.y - 50 };
+    getPositionInPaperCoords(startEventPosition).then(({ x, y }) => {
+      getElementAtPosition(topLanePosition, nodeTypes.poolLane).click();
+
+      cy.get('.main-paper [button-id="bottom-right-resize-button"]')
```

Comment:
Nice. I do see that there can be two bottom-right-resize-buttons. But it seems to work regardless.
![Screen Shot 2019-11-21 at 2 33 31 PM](https://user-images.githubusercontent.com/6653340/69370530-1e258c80-0c6c-11ea-8a32-460f51ba1511.png)

Reply:
How did you search for the 2nd bottom right resize button? It seems to not be visible in the screenshot above. I think it may be part of the mini-map paper, which I think duplicates the handles as well.

Reply:
I highlighted the pool and then did a text search in the Elements console. The other bottom right resize was where I expected it to be.

You are correct. It is the mini-map!